sorc/chgres_cube.fd/input_data.F90

```diff
@@ -6639,4 +6640,27 @@ subroutine check_soilt(soilt, landmask, skint)
     enddo
   enddo
 end subroutine check_soilt
+
```

Comment:
As an FYI, the 0.5 is the maximum value used by the Noah LSM. And it is in millimeters. I would add the units to the documentation.